addons/website/static/src/scss/secondary_variables.scss

```diff
@@ -69,8 +69,14 @@ $o-website-values-palettes: append($o-website-values-palettes, $-actual-user-web
 // Enable last website values palette, which is now the user customized one
 $o-website-values-palette-number: length($o-website-values-palettes);
 $o-website-values: $-actual-user-website-values-palette !default;
-@function o-website-value($key) {
-    @return map-get($o-website-values, $key);
+@function o-website-value($key, $fallback: null) {
+    @if map-get($o-website-values, $key) {
+        @return map-get($o-website-values, $key);
+    } @else if $fallback {
+        @return $fallback;
+    } @else {
+        @return null;
+    }
```

Comment:
This is equivalent to `@return map-get($o-website-values, $key) or $fallback` (except in your case where null is returned if you give a falsy fallback, which is not consistent). But even then, that's not ok. If the website-value you request is "false", you don't expect the fallback to be returned.

In the end, I would not change anything about this, at least not for the usecases I see here:

```scss
$form-range-track-height: o-website-value('form-range-track-height', .1rem) !default;
```
```scss
'form-range-track-height': null, // Default to BS
```

(first: your comment is copy/pasted: it does not default to BS since you force another value in case no value is configured. I guess we should remove all those "default to BS" comments to be honest, may be confusing)

What I would do is remove the fallback in website/bootstrap_overriden and set that fallback in web/bootstrap_overridden_frontend (and leave the default in the base palette to "null" as you did but without BS comment). That way:
- You change the design of all frontend components, whether or not website is installed (as you should), as a default customization of bootstrap for the frontend
- Once website is installed, if a value is configured by the theme/user, it overrides the default one

In the end the fact that some BS variables are forced to a default value in website/bootstrap_overridden is a mistake, that we are fixing once we can.

Reply:
Ok thanks....  indeed, I wasn't sure about this... like at all ^^